source/octf/proto/InterfaceTraceParsing.proto

```diff
@@ -28,6 +29,15 @@ message ParseTraceRequest {
     ];
 }
 
+message GetTraceStatisticsRequest {
+    string tracePath = 1 [
```

Comment:
Unfortunately there is not such mechanism, and input is a message with primitive fields types. But don't worry, here it can defined different field like: time unit, specific device, IO class, etc. So won't be duplicated.